### ISSUE TITLE
Broken link in FAQ + dummy text in Research URL

### ISSUE THREAD

**Issue by joe-brilliant** [globaldothealth/list#1725]
Link to GitHub on FAQs under "schema" to this link results in 404

https://github.com/globaldothealth/list/blob/main/data-serving/scripts/export-data/case_fields.yaml 

https://www.dropbox.com/s/9a6gh5s5wsp6ah7/Screen%20Shot%202021-04-07%20at%207.14.26%20PM.png?dl=0 

URL string for this page on research still has dummy text:

https://global.health/2020/06/16/neque-porro-quisquam-est-qui-dolorem-ipsum-quia-dolor-sit-amet/

**Comment by rkassa** [comment]
@joe-brilliant @abhidg 

Schemas should be going to this URL, right?
https://github.com/globaldothealth/list/tree/main/data-serving/data-service/schemas

**Comment by joe-brilliant** [comment]
I believe that's right, thanks @rkassa ! @abhidg can you confirm?

**Comment by abhidg** [comment]
Yes, specifically https://github.com/globaldothealth/list/blob/main/data-serving/data-service/schemas/cases.schema.json

**Comment by joe-brilliant** [comment]
@rkassa would you mind taking a look at the url text? Thanks!

**Comment by rkassa** [comment]
The dummy-latin-text text in the URL aliases is resolved on both dev and prod
